### ISSUE TITLE
NEW VERSION OF ANE'S SHOWS DEMO POPUP EVEN AFTER REGISTERING IN ANELAB

### ISSUE THREAD

**Issue by Gokulv617** [myflashlab/ANE-LAB#160]
The latest version of ANE's shows demo popup even after registering in ANELAB. When I click register button it shows "done" dialog pop. The anelab id is also present in the xml. But when I install in mobile device iphone it shows demo ane popup. Please fix this.

![img_20210605](https://user-images.githubusercontent.com/43432782/120878329-46b87180-c5d9-11eb-9954-35495af961ef.png)

**Comment by Gokulv617** [comment]
is this fixed ??

**Comment by Gokulv617** [comment]
please reply ?

**Comment by Gokulv617** [comment]
ANELAB cannot recognize some of the new ANE's,

- Admob V6.2.4
- Billing V5.0.0
- Gameservices V6.1.5
- GoogleSignIn V5.2.5

![img_20210617](https://user-images.githubusercontent.com/43432782/122352452-c563c600-cf6c-11eb-95b5-5d2c41583d36.png)

**Comment by Gokulv617** [comment]
@nekofar @myflashlab 

Does someone else face this problem ? I'm struggling with this from past 12 days.

**Comment by Gokulv617** [comment]
Please let me know whether you guys will provide support or not ? So I can move my apps to distriqt ANE's 

It's been 16 days since I raised my query, but still no one to address it.

**Comment by Gokulv617** [comment]
is this solved ?

**Comment by joypalsingh** [comment]
> is this solved ?

I don't think they are going to fix any issues now as they announced the end of MyFlashLabs services (https://www.myflashlabs.com/sunset/)

**Comment by Gokulv617** [comment]
Yes I got the mail just before. They can at least fix this issue before leaving. Their updated ANE's are not gonna work if it is not fixed.

**Comment by joypalsingh** [comment]
> Yes I got the mail just before. They can at least fix this issue before leaving. Their updated ANE's are not gonna work if it is not fixed.

Ye, it would be great if they could do that, I am also trying to contact the support for more than 1 month but received no reply till now.

**Comment by Gokulv617** [comment]
I'm also waiting for their reply. It seems they doesn't want to reply to our queries now

**Comment by Gokulv617** [comment]
@myflashlab @nekofar 

Please fix this issue before shutting down. It will give us some more time to transfer our ANE's with other provider.

**Comment by Gokulv617** [comment]
I also found the new FirebaseCore ANE v11.0.1 doesn't take the config files and doesn't initialize. So Firebase analytics and Cloud messaging is failing.

Error msg: _"Config file not found"_
